standards/technical-approach.md

```diff
@@ -16,24 +36,56 @@ TADs do not have to be created for all development work, but MUST be created in
+
+## TAD Template
+
+The TAD template can be found in the Google Drive NYPL template gallery. Select it in Google Docs using `File > New > From template gallery` and make sure you're on the `NYPL` tab. Navigate to the `NYPL` section header and the TAD template will be in the list.
+
+The TAD template includes several sections that serve as a guide for getting started. Feel free to add or reformat sections as you see fit. Take a look at the [examples](#tad-examples) below for inspiration.
+
+Writers should start by stating the problem and goals in a couple of sentences or paragraphs and then add an architectural summary of the problem at hand as it relates to the existing code base or infrastructure. For features that involve deep understanding of the underlying infrastructure and/or changes to an architectural pattern, this section can be quite long and include many sub-sections. See the [DAM — Virtual Reading Room, Phase 1A TAD](https://docs.google.com/document/d/1c9f2lxb8-ANWdMxcLIej_bN87muSfP7WkLs7zS3BWOc/edit#heading=h.kutc298lyner) as an example.
+
+If there are multiple options to weigh before proposing a final recommendation, each option should include a description and a list of pros and cons.
+
+Writers should consider multiple layers of the technical stack in the TAD: backend, frontend, APIs, infrastructure (devops), and security concerns, for example. If you're unsure about any of these sections, reach out for help from your Tech Lead or engineering peers for help. Talking with other teams, i.e. devops, frontend, or backend, is also encouraged throughout the writing process.
+
+The recommendations section should clearly state the final recommended approach alongside a summary or list of any other recommendations discussed throughout the TAD.
+
+The next steps section can be a short summary or list of next steps that need to be taken. This section does not need to include links to Jira tickets.
 
 ## Process
 
 All TADs MUST follow this process to be approved, at which point JIRA tickets can be created and development work can start.
 
-1. The most senior developer on the feature/task/story is assigned a ticket to write the TAD and the ticket MUST be tagged with `tech-approach`
-2. The assigned developer should collect as much information from the BRD, stakeholders and any other relevant information sources and write the TAD
-3. The TAD MAY be circulated at this point for feedback within a team or stakeholders for further input/feedback
+1. An engineer is assigned a ticket to write the TAD and the ticket MUST be tagged with `tech-approach`. Assigning points to a TAD-writing ticket can be challenging. Generally a TAD is given 5 or 8 points, depending on the complexity of the feature and if a [proof of concept](#proof-of-concepts) is necessary.
```

Comment:
@oliviawongnyc added some detail here about pointing TADs. The recent PJR TADs that I have seen are all pointed at 5 points, which seems standard, but I don't think 8 is unreasonable.